### ISSUE TITLE
First run: KeyError: 'albumNameEnc'

### ISSUE THREAD

**Issue by abhsc** [skarppi/icloud_photo_station#12]
Package built just fine and installed to the NAS (DS418play, DSM 6.2.3-25426 Update 2)

I have 2FA on my account and ran the command in an SSH session, the 2FA worked, but the command then failed. This is the error I received when issuing the command within the SSH session: 

```
$ python /volume1/@appstore/icloud_photo_station/app/icloudpd.py  --username 'itunes_user@domain.com' --password 'pass' --auto-delete --recent 10  --album 'Recents'
2020-10-09 09:44:21 DEBUG    Authenticating...
Traceback (most recent call last):
  File "/volume1/@appstore/icloud_photo_station/app/icloudpd.py", line 5, in <module>
    main()
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/volume1/@appstore/icloud_photo_station/app/icloudpd/base.py", line 261, in main
    photos = icloud.photos.albums[album]
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/pyicloud_ipd/services/photos.py", line 194, in albums
    folder['fields']['albumNameEnc']['value']).decode('utf-8')
KeyError: 'albumNameEnc'
```

I have tried no album, no dice. Please advise

**Comment by cmfrazier** [comment]
Same error for me.
```
2021-01-31 16:14:53 DEBUG    Authenticating...
Traceback (most recent call last):
  File "/volume1/@appstore/icloud_photo_station/app/icloudpd.py", line 5, in <module>
    main()
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/volume1/@appstore/icloud_photo_station/app/icloudpd/base.py", line 261, in main
    photos = icloud.photos.albums[album]
  File "/volume1/@appstore/icloud_photo_station/env/lib/python2.7/site-packages/pyicloud_ipd/services/photos.py", line 194, in albums
    folder['fields']['albumNameEnc']['value']).decode('utf-8')
KeyError: 'albumNameEnc'
```

**Comment by cmfrazier** [comment]
seems to be related here > https://github.com/icloud-photos-downloader/icloud_photos_downloader/issues/150

**Comment by skarppi** [comment]
This was fixed in the upstream repository and now included in the 0.3.0 release for DSM7 or ```dsm6``` branch for DSM6.
